src/accelerate/__init__.py

```diff
@@ -16,7 +15,18 @@
     InitProcessGroupKwargs,
     find_executable_batch_size,
     infer_auto_device_map,
+    is_torch_version,
     load_checkpoint_in_model,
     rich,
     synchronize_rng_states,
 )
+
+
+if is_torch_version(">=", "1.9.0"):
+    from .big_modeling import (
+        cpu_offload,
+        disk_offload,
+        dispatch_model,
+        init_empty_weights,
+        load_checkpoint_and_dispatch,
+    )
```

Comment:
It should be possible to import those regardless of the version (and then the actual use of it would trigger the error).

Reply:
Fixed